.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -3,24 +3,27 @@ description: Propose an enhancement for Babelfish for PostgreSQL Website
 title: "[Enhancement]: "
 labels: ["enhancement", "untriaged"]
 assignees:
-  - amazon-auto
+  - 3manuek
+  - gopinathpai
+  - xaviersierra
+  - nasbyj
 body:
   - type: markdown
     attributes:
       value: |
-        Thanks for taking the time to fill out this bug report!
+        Thanks for taking the time to fill out this feature request.
   - type: input
     id: contact
     attributes:
       label: Contact Details
-      description: How can we get in touch with you if we need more info?
+      description: Please enter your mail address for contacting in the case of needing more information.
       placeholder: ex. email@example.com
     validations:
       required: false
   - type: textarea
     id: what-intends
     attributes:
-      label: What this feature/enhancement tries to solve?
+      label: Describe what this feature intends to solve.
```

Comment:
How about:

Describe how this change will improve Babelfish.

Reply:
I want to emphasize that the _enhacement_ template is for the website specifically. @susanmdouglas-aws  WDYT about:

> Describe how this change will improve Babelfish website.